src/fdt.rs

```diff
@@ -114,6 +114,21 @@ impl Fdt {
 		Ok(self)
 	}
 
+	#[cfg(target_arch = "aarch64")]
+	pub fn timer(mut self) -> FdtWriterResult<Self> {
+		let node_name = "timer";
+
+		let timer_node = self.writer.begin_node(node_name)?;
+		self.writer
+			.property_array_u32("interrupts", &[1, 13, 4, 1, 14, 4, 1, 11, 4, 1, 10, 4])?;
```

Comment:
I think, a comment explaining this field would save future coders a lot of head scratching :wink:

Reply:
I added a comment.